tensorflow/contrib/BUILD

```diff
@@ -65,6 +65,7 @@ py_library(
         "//tensorflow/contrib/linalg:linalg_py",
         "//tensorflow/contrib/linear_optimizer:sdca_estimator_py",
         "//tensorflow/contrib/linear_optimizer:sdca_ops_py",
+	"//tensorflow/contrib/lms:lms_py",
```

Comment:
Make consistent indent?

Reply:
I don't see indent went wrong when opening the file by emacs or vim. Could you please confirm again?

Reply:
Oh, I see. Thanks.

Reply:
I think the indent was done via a tab, while all the other lines were done via spaces. It probably should be spaces to be consistent.

Reply:
@wdirons thanks. I will check and change to spaces.